x-pack/plugin/esql/compute/src/main/java/org/elasticsearch/compute/data/DocBlock.java

```diff
@@ -84,19 +84,19 @@ public void closeInternal() {
     /**
      * A builder the for {@link DocBlock}.
      */
-    public static Builder newBlockBuilder(int estimatedSize, BlockFactory blockFactory) {
-        return new Builder(estimatedSize, blockFactory);
+    public static Builder newBlockBuilder(BlockFactory blockFactory, int estimatedSize) {
```

Comment:
Do we need to switch the arguments here? This is now inconsistent with `ElementType.newBlockBuilder`.

Reply:
Yes, to be consistent with other Block.Builder methods, I can follow up to switch the parameters for ElementType.